docusaurus/docs/iOS/guides/getting-started.md

```diff
@@ -1,3 +1,308 @@
 ---
 title: Getting Started
----
\ No newline at end of file
+---
+
+As step 0, we recommend visiting [our webpage](getstream.io) and creating an account. You'll need to create an account, and acquire an [API key](https://getstream.io/try-for-free/) to be able to use our platform.
+
+## Overview
+
+StreamChat Swift SDK consists of two separate frameworks
+
+- `StreamChat` is the low-level client that provides the main chat functionality including offline storage and optimistic updates. You can use it directly in case you want to build your own UI layer for the chat.
+
+- `StreamChatUI` is the `UIKit` and `SwiftUI` framework that provides the complete set of reusable and customizable UI components for the common chat experience in iOS apps. It uses `StreamChat` under the hood. Unless your UI is completely different from the common industry standard, you should be able to customize the built-in components to match your needs.
+
+We suggest using `StreamChatUI` for most of our users.
+
+## SDK Basics
+
+The `StreamChat` framework has just three main types of components
+
+- `ChatClient` is the center point of the SDK. It represents the Stream Chat service. For the absolute majority of the use cases, you will need just a single instance of `ChatClient` in your app.
+
+- `xxxController` objects are lightweight and disposable objects that allow you to interact with entities in the chat system. All controllers are created using a `ChatClient` object. See below for more info about controllers.
+
+- Model objects like `ChatUser`, `ChatChannel`, `ChatMessage`, etc. are lightweight immutable snapshots of the underlying chat objects at the given time. You can access the model objects anytime via its respective controller counterpart.
+
+:::note
+If you're using `StreamChatUI` SDK, you don't need to know much about `StreamChat` Controllers. The UI SDK handles interactions with Controllers.
+:::
+
+### StreamChat Controllers
+
+The most typical interaction with the StreamChat SDK is asking `ChatClient` for a controller and using it to get/observe data.
+
+#### Controllers can have a very short lifespan and can be used for simple mutations
+
+Controller were designed as lightweight disposable objects. You can quickly create them, perform a mutation on the underlying entity, and throw them away:
+```swift
+chatClient
+  .channelController(for: <ChannelId>)
+  .createNewMessage(text: "My first message")
+```
+
+#### Controllers can be used for continuous observations of the given object
+
+Controllers can also act as entity observers and monitor changes of the represented entity. There's no limitation in terms of how many controllers can observe the same entity.
+
+You can choose the preferred way you want to be notified about the changes:
```

Comment:
We should try not to present more options than necessary in getting started. Are there specific use cases for these different approaches? 

If yes, could you add a line or two summarizing when / why to use one of the example approaches over another.

Reply:
It's actually pretty clear for iOS devs but I'm adding a couple of sentences.